src/pages/contacts/index.vue

```diff
@@ -2,11 +2,15 @@
   <div class="contactsPage dappPageWrapper">
     <i-modal id="create-edit-contact" v-model="contactModal.enabled" class="prevent-close" size="lg">
       <button id="contactBackBtn" @click="contactModal.enabled = false">
-        <img src="../../static/images/back_icon.svg" /> Back
+        <img v-if="$inkline.config.variant == 'dark'" src="../../static/images/back_icon_white.svg" />
+        <img v-if="$inkline.config.variant !== 'dark'" src="../../static/images/back_icon.svg" />
```

Comment:
could this line just be `v-else` instead of `v-if="$inkline.config.variant !== 'dark'"`?